dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Machines/VoltageRegulators/Standard/ExcIEEEST4B_INIT.mo

```diff
@@ -64,8 +62,8 @@ equation
     Line(points = {{-140, 0}, {-100, 0}, {-100, 36}, {-82, 36}}, color = {85, 170, 255}));
   connect(ut0Pu, potentialCircuit.uT) annotation(
     Line(points = {{-140, 80}, {-100, 80}, {-100, 44}, {-82, 44}}, color = {85, 170, 255}));
-//  connect(rectifierRegulationCharacteristic.y, product.u2) annotation(
-//    Line(points = {{41, -40}, {60, -40}, {60, -6}, {77, -6}}, color = {0, 0, 127}));
+  connect(rectifierRegulationCharacteristic.y, product.u2) annotation(
```

Comment:
Do they nrt reference need to be updated ?

Reply:
No, the nrt have been successfully run.